src/cache_invalidate.c

```diff
@@ -75,64 +72,6 @@ cache_invalidate_callback(Datum arg, Oid relid)
 		hypertable_cache_invalidate_callback();
 }
 
-static inline CmdType
-trigger_event_to_cmdtype(TriggerEvent event)
-{
-	if (TRIGGER_FIRED_BY_INSERT(event))
-		return CMD_INSERT;
-
-	if (TRIGGER_FIRED_BY_UPDATE(event))
-		return CMD_UPDATE;
-
-	return CMD_DELETE;
-}
-
-PGDLLEXPORT Datum invalidate_relcache_trigger(PG_FUNCTION_ARGS);
-
-TS_FUNCTION_INFO_V1(invalidate_relcache_trigger);
-
-/*
- * Trigger for catalog tables that invalidates caches.
- *
- * This trigger generates a cache invalidation event on changes to the catalog
- * table that the trigger is defined for.
- */
-Datum
-invalidate_relcache_trigger(PG_FUNCTION_ARGS)
-{
-	TriggerData *trigdata = (TriggerData *) fcinfo->context;
-
-	if (!CALLED_AS_TRIGGER(fcinfo))
-		elog(ERROR, "not called by trigger manager");
-
-	catalog_invalidate_cache(RelationGetRelid(trigdata->tg_relation),
-							 trigger_event_to_cmdtype(trigdata->tg_event));
-
-	/* tuple to return to executor */
-	if (TRIGGER_FIRED_BY_UPDATE(trigdata->tg_event))
-		return PointerGetDatum(trigdata->tg_newtuple);
-	else
-		return PointerGetDatum(trigdata->tg_trigtuple);
-}
-
-TS_FUNCTION_INFO_V1(invalidate_relcache);
-
-/*
- * Force a cache invalidation for a catalog table.
```

Comment:
We found this function useful in the past for debugging with users. I am not sure we should remove.